nexus/src/app/lldp.rs

```diff
@@ -103,13 +103,8 @@ impl super::Nexus {
         };
         neighbors.sort_by_key(|n| n.id);
 
-        let mut limit = usize::try_from(limit)
+        let limit = usize::try_from(limit)
```

Comment:
no need to change this, but fwiw I've mostly seen this as a `as usize`.

Reply:
This statement threw @jgallagher into a form of apoplexy I've rarely seen. While my statement is factually accurate in every way, I insist that you **not** change this!

Reply:
For a tiny bit of context since this is a little out of left field and very pedantic (sorry!) - assuming `limit` is a u32, `limit as usize` is technically fine since we're never going to run Nexus on a 16-bit CPU where that conversion would be silently lossy. But - I prefer to stay away from `as` even in cases where it's fine like this, because there are other cases where it's definitely not fine (e.g., `some_usize as u32`). I don't love the verbosity of `usize::try_from(...).unwrap()` at all, but still prefer to `as`, personally.

Reply:
This should always work on any system on which nexus will run, so both `as` and `unwrap` should work.  In this kind of scenario, I go with `expect`, both in case I'm wrong and it does blow up un`expect`edly, and as a comment to reviewers and subsequent readers of the code.